docs/src/components/pricing/PricingTable.tsx

```diff
@@ -457,7 +457,7 @@ const rowHeaders: Record<string, React.ReactNode> = {
     <ColumnHead label="Column filters" nested href="/x/react-data-grid/filtering/" />
   ),
   'data-grid/filter-quick': (
-    <ColumnHead label="Quick filter" nested href="/x/react-data-grid/filtering/quick-filter/" />
+    <ColumnHead label="Search" nested href="/x/react-data-grid/filtering/quick-filter/" />
```

Comment:
Not as long as the filter model property is called `quickFilterValues`

https://mui.com/x/react-data-grid/filtering/quick-filter/

If we rename this, it might be in v7

Reply:
Instead of renaming it, what do you think about changing it to `Quick filter (Search)`?
So that we capture both "quick filter" and "search" keywords in the docs, without impacting the API.

On a related note, this also reminds me of the [Search feature](https://github.com/mui/mui-x/issues/5082) that we might want to build in the future, in which case we will have a naming collision 🙂

Reply:
> we will have a naming collision

@cherniavskii Oooooh yeah. Our "Quick filter" feature feels more of a filter than a true search feature. 👍 to:

- Rename to "Quick filter (Search)" now
- Use "Search" for #5082 or maybe "Cell search"